core/build.gradle

```diff
@@ -101,7 +101,7 @@ dependencies {
   compile 'com.android.support:appcompat-v7:24.2.1'
 
   compile 'com.mapzen.android:lost:2.1.2'
-  compile 'com.mapzen.android:pelias-android-sdk:1.0.0'
+  compile 'com.mapzen.android:pelias-android-sdk:1.1.0-SNAPSHOT'
```

Comment:
Yes indeed once all changes are merged to `pelias-android-sdk` and `on-the-road` we will deploy a new release of each and remove the snapshot tags here.